### DISCUSSION TITLE
Are you sure you configured your project correctly so it uses a the pgservice configuration

### DISCUSSION THREAD

**Discussion by yaoyuandejuli** [opengisch/QField#3016] [Q&A]
Hello ,

I am quite sorry but looking a the tutorials it is hard for me to understand what am i supposed to do with pgservice configuration thingy. 

Let s say i am creating and uploading a project on qfieldcloud using a direct database connection with multiple users. 
->'if your QField users will not have a reliable internet connection in the field or Direct database access'

Ok i can do all that what i can't figure out is what do i do with this mysterious pg_config file , do I just create a secret or do i need to do something physically with a text files what is going on . I have managed to have project on qfield cloud working directly from the postgres database without doing this but other time i get errors. 

If somebody kind enough could lay out the steps in details for a novice to undertake it would be greatly appreciated . 

1.create QGIS project
2.Add postgres layer 
3.Set up direct database connection 
4.upload project  to qfield cloud 
5. ????
6. ????

Kind regards ,

Hugo

**Comment by lucienicolier** [comment]
Hello Hugo, 

Did you have a look to this part of the docs?

[PostGIS](https://docs.qfield.org/get-started/tutorials/advanced-setup-qfc/#postgis)

Best regards,

Lucie

**Reply by yaoyuandejuli** [reply]
Bonjour Lucien,

Thanks for your reply, Yes I read those tutorial carefully . 
So I did multiple projects with direct connection it was working great for me with the layer updating in real time etc .... 

Then i added a collaborator and he got the project using qfieldcloud on his android . So it should be the exact same project that i ve got on my phone . 
But he got the following error message 

![image](https://user-images.githubusercontent.com/95315172/176472882-5c931a46-fad4-4263-8e41-9b34c2e8d87f.png)

I deleted the project on my phone redownload it through qfield cloud and got the same error msg . the project on the laptop is still using the postgis layer , I dont get it . 

Regards,

Hugo

**Reply by lucienicolier** [reply]
@suricactus do you have an idea what is wrong? Thanks

**Comment by GaryHsiao88** [comment]
I got the same problem yesrterday.
And then I tried to make the version of QField and QGIS on PC or laptop same. 
(I installed latest QGIS version 3.30.1-'s-Hertogenbosch)
Finally, upload project again and it works.
